content/SCALE/SCALETutorials/Shares/AddingNFSShares.md

```diff
@@ -64,15 +60,17 @@ Leave the field empty to allow all systems access to the share.
 
 If you want to tune the NFS share access permissions or define authorized networks, click **Advanced Options**.
 
-![AddNFSAdvancedOptionsAccessSettings](/images/SCALE/22.12/AddNFSAdvancedOptionsAccessSettings.png "Add NSF Advanced Options Access Settings")
+{{< trueimage src="/images/SCALE/22.12/AddNFSAdvancedOptionsAccessSettings.png" alt="Add NSF Advanced Options Access Settings" id="3 Add NSF Advanced Options Access Settings" >}}
 
 Select **Read-Only** to prohibit writing to the share. 
 
-To map user permissions to the *root* user, enter a string or select the user from the **Maproot User** dropdown list. To map the user permissions to all clients, enter a string or select the user from the **Mapall User** dropdown list.
+To map user permissions to the root user, enter a string or select the user from the **Maproot User** dropdown list. 
+To map the user permissions to all clients, enter a string or select the user from the **Mapall User** dropdown list.
 
-To map group permissions to the *root* user, enter a string or select the group from the **Maproot Group** dropdown list. To map the group permissions to all clients, enter a string or select the group from the **Mapall Group** dropdown list.
+To map group permissions to the *root* user, enter a string or select the group from the **Maproot Group** dropdown list. 
```

Comment:
"root user" should be consistently either italicized or not. See line 67

Reply:
actually in this instance the root user is not a variable, it is talking about the root user. We've had issues with inconsistent style emphasis on this where sometimes they apply the command emphasis when talking about root user. The question is, am I using root as a possible user in the context of an example, or am I talking about what to do for the root user. I removed the *root* but missed it in the second instance.